File: tests/fixtures/sample.test.js
// Sample JavaScript test file

function testFunction() {
    console.log("This is a test function");
    expect(true).toBe(true);
}

const largeTestFunction = () => {
    // Large test function
    const a = 1;
    const b = 2;
    const c = 3;
    console.log("test 1");
    console.log("test 2");
    console.log("test 3");
    console.log("test 4");
    expect(a + b).toBe(c);
};

File: tests/fixtures/SampleTest.java
// Sample Java test file
import org.junit.Test;

public class SampleTest {
    
    @Test
    public void testMethod() {
        System.out.println("This is a test");
        assert true;
    }
    
    @Test
    public void largeTestMethod() {
        // This is a large test method
        int a = 1;
        int b = 2;
        int c = 3;
        System.out.println("test 1");
        System.out.println("test 2");
        System.out.println("test 3");
        System.out.println("test 4");
        assert a + b == c;
    }
}
